terraso_backend/apps/graphql/schema/__init__.py

```diff
@@ -30,7 +30,13 @@
     MembershipNode,
     MembershipUpdateMutation,
 )
-from .users import UserAddMutation, UserDeleteMutation, UserNode, UserUpdateMutation
+from .users import (
+    UserAddMutation,
+    UserDeleteMutation,
+    UserNode,
+    UserPreferenceUpdate,
```

Comment:
question: does this handle add, update and delete?

Reply:
@paulschreiber not delete, but add and update it does, I can add a delete mutation too